packages/trader/src/sass/app/modules/statement.scss

```diff
@@ -0,0 +1,33 @@
+.statement {
+    &__accountStatistics {
+        max-width: 33.3rem;
+        width: 100%;
+        height: 6.4rem;
+        display: flex;
+        flex-direction: row;
+        text-align: left;
+
+        &--Rectangle {
+            text-align: left;
+            height: 100%;
+            border-left: 1px solid $color-grey-2;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            padding: 8px 16px;
+            align-items: flex-start;
+
+            &--title {
```

Comment:
BEM modifiers can't have children themselves. Please consider another naming strategy for `--Rectangle` or flatten out below classes.